mods/ra/weapons/smallcaliber.yaml

```diff
@@ -272,8 +268,8 @@ M60mg:
 	Report: gun5.aud
 	ValidTargets: Ground, Infantry
 	InvalidTargets: Vehicle, Water, Structure, Wall, Husk, Mine
-	Projectile: Bullet
-		Speed: 1c682
+	Projectile: InstantHit
+		Blockable: true
```

Comment:
Is this change intentional? I don't think you explained it in the description.

Reply:
Ok, I see. `Bullet` defaults to `Blockable: true` and `InstantHit` to `Blockable: false`.

Reply:
Yeah, it's strange in the way defaults are set up. But kinda makes sense in the way ORA mods work